### PR TITLE
feat: Added back Button on operator settings & edit product details page

### PR DESCRIPTION
**Pull request by NitishAroraKPMG**
## Description

Added the back button for the operator settings and the pages navigated via the edit button from product details

## Testing instructions

Edit the settings in operator settings 
Click on edit button in product details page and the Back button will appear on the pages.

### CODE REVIEW

**Comment by alainajaffer** [comment]
/multiOperatorServiceList needs a back for multioperator products

File: repos/fdbt-site/src/pages/manageOperatorGroup.tsx [hunk 1]
```diff
@@ -1,74 +0,0 @@
-import React, { ReactElement } from 'react';
-import CsrfForm from '../components/CsrfForm';
```
Comment (RobCattonKPMG) [inline repos/fdbt-site/src/pages/manageOperatorGroup.tsx:2]: How come we deleted this? Is it no longer needed?

File: repos/fdbt-site/src/interfaces/index.ts [hunk 2]
```diff
@@ -131,6 +131,7 @@ export interface UserDataUploadsProps {
     showPriceOption?: boolean;
     poundsOrPence?: string | null;
     csrfToken: string;
+    backHref?: string;
```
Comment (RobCattonKPMG) [inline repos/fdbt-site/src/interfaces/index.ts:134]: can we remove the ?

File: repos/fdbt-site/src/pages/selectTimeRestrictions.tsx [hunk 3]
```diff
@@ -17,16 +23,19 @@ interface SelectTimeRestrictionsProps {
...
 const SelectTimeRestrictions = ({
     csrfToken,
     errors,
     timeRestrictions,
     selectedId,
+    backHref,
 }: SelectTimeRestrictionsProps): ReactElement => {
     return (
         <FullColumnLayout title={title} description={description} errors={errors}>
+            {!!backHref && errors.length === 0 ? <BackButton href={backHref}></BackButton> : null}
```
Comment (RobCattonKPMG) [inline repos/fdbt-site/src/pages/selectTimeRestrictions.tsx:38]: Can we change all of the backbuttons to be   <BackButton href={backHref} /> (self closing)

Comment (NitishAroraKPMG) [reply repos/fdbt-site/src/pages/selectTimeRestrictions.tsx:38]: ok ... will do that

**Review by RobCattonKPMG — APPROVED** [review]

**Pull request merged**